Source/KerbalCombatSystems/Debug.cs

```diff
@@ -30,6 +35,7 @@ private void Update()
                 //switch bool return
                 showLines = !showLines;
 
+                //this block strikes me as redundant to line 91 but not actively harmful
```

Comment:
```suggestion

```
It hides unused lines that would otherwise linger until they get hidden by the line cleaner.

Reply:
Understood